### ISSUE TITLE
Any chance this can work with cordova-ios 4.5.5?

### ISSUE THREAD

**Issue by jamauro** [AyogoHealth/cordova-plugin-wkkeyboardfix#1]
Thanks for putting this fix together. I'm pinned to cordova-ios `4.5.5` because my app runs on Meteor. Is there any way to use this fix on that version?

**Comment by dpogue** [comment]
Probably. I set the required version to cordova-ios >=5.0.0 because that was the only one I was totally confident it would work with, and I didn't have any projects using older versions of cordova-ios to test with. However, there's no reason the plugin shouldn't be compatible with 4.5.x versions.

I've pushed a commit to a branch that lowers the required version, if you are willing to give it a test and report back if it works or not. I believe this command should work for installing:
```
cordova plugin add "github:AyogoHealth/cordova-plugin-wkkeyboardfix#4.5-compat"
```

**Comment by dpogue** [comment]
@jamauro Any luck testing with cordova-ios 4.5.x? I'm happy to release a version with less strict version requirements if someone can confirm that it actually works 😅

**Comment by jamauro** [comment]
Will test tonight. Thanks!

**Comment by jamauro** [comment]
@dpogue looks good! Thanks. Would really appreciate you updating for a less strict version requirements. 

Any idea when the bug might be patched in WKWebView? I know you submitted the fix but not sure if they give any indication when it might be merged.

Also in case anyone else stumbles across this, to add it to meteor I used `meteor add cordova:cordova-plugin-wkkeyboardfix@https://github.com/AyogoHealth/cordova-plugin-wkkeyboardfix.git#9e481af91be466b3233d480082bb81d46075c69e`

**Comment by dpogue** [comment]
Thanks! I've published cordova-plugin-wkkeyboardfix@1.0.1 which includes support for cordova-ios@4.5.x
